get_fm_data.py

```diff
@@ -0,0 +1,200 @@
+from datetime import datetime, timedelta
+import json
+import pytz
+import re
+import requests
+import time
+from typing import AsyncGenerator, List, Optional
+
+import appdaemon.plugins.hass.hassapi as hass
+import isodate
+
+
+class FMdata(hass.Hass):
```

Comment:
Rename class to something more explicit, for example, `PriceImporter`?

Reply:
Renamed to data_importer (as it will hopefully in the future als import revenue data).
For this to work the file apps.yaml also needs an update for this name change.
get_fm_data:
  module: get_fm_data
  class: data_importer

Reply:
I'm moving this refactoring work to a separate PR (see https://github.com/SeitaBV/v2g-liberty/issues/26), to avoid merge conflicts with other ongoing PRs.